include/dlaf/eigensolver/eigensolver.h

```diff
@@ -0,0 +1,31 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2022, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+#pragma once
+
+#include <blas.hh>
+#include "dlaf/eigensolver/eigensolver/mc.h"
+#include "dlaf/matrix/matrix.h"
+#include "dlaf/types.h"
+#include "dlaf/util_matrix.h"
+
+namespace dlaf {
+namespace eigensolver {
+
+/// TODO
```

Comment:
TODO what?

Reply:
As TODO is preceded by 3 `\` it refers to the missing documentation... 😁

Reply:
Ah, thanks :heavy_check_mark: